banderwagon/element.go

```diff
@@ -170,9 +171,15 @@ func BatchToBytesUncompressed(elements ...*Element) [][UncompressedSize]byte {
 }
 
 func (p *Element) setBytes(buf []byte, trusted bool) error {
+	if len(buf) != CompressedSize {
+		return errors.New("invalid compressed point size")
+	}
```

Comment:
Found by fuzzer. The reason this got detected is because if `buf` is empty, then gnark-crypto interpret that as `x == 0` which is a valid point. But if you serialize the identity point, you get a different serialization than the original.

That's to say: a compressed point being represented by an empty byte stream was considered valid.